data/soapvalidator/Prefs/OutOfOffice/OOOPreventive.xml

```diff
@@ -0,0 +1,192 @@
+                    <password>${defaultpassword.value}</password>
+                </CreateAccountRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:CreateAccountResponse/admin:account" attr="id" set="account1.id"/>
+                <t:select path='//admin:CreateAccountResponse/admin:account/admin:a[@n="zimbraMailHost"]' set="account1.server"/>
+            </t:response>
+        </t:test>
+
+        <!-- Account 2 Creation -->
+        <t:test id="create_account2" required="true" depends="admin_login">
+            <t:request>
+                <CreateAccountRequest xmlns="urn:zimbraAdmin">
+                    <name>${account2.name}</name>
+                    <password>${defaultpassword.value}</password>
+                </CreateAccountRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//admin:CreateAccountResponse/admin:account" attr="id" set="account2.id"/>
+                <t:select path='//admin:CreateAccountResponse/admin:account/admin:a[@n="zimbraMailHost"]' set="account2.server"/>
+            </t:response>
+        </t:test>
+    </t:test_case>
+
+    <!-- Set OOO message for Account 1 -->
+    <t:test_case testcaseid="SetOOO" type="always" depends="AdminSetup">
+        <t:objective>Account 1 sets Out Of Office message</t:objective>
+        
+        <t:property name="server.zimbraAccount" value="${account1.server}"/>
+        
+        <t:test id="auth_account1" required="true">
+            <t:request>
+                <AuthRequest xmlns="urn:zimbraAccount">
+                    <account by="name">${account1.name}</account>
+                    <password>${defaultpassword.value}</password>
+                </AuthRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//acct:AuthResponse/acct:authToken" set="authToken"/>
+            </t:response>
+        </t:test>
+        
+        <!-- Enable OOO with From and untilDate in the past -->
+        <t:test id="set_ooo_until_date" required="true" depends="auth_account1">
+            <t:request>
+                <ModifyPrefsRequest xmlns="urn:zimbraAccount">
+                    <pref name="zimbraPrefOutOfOfficeReplyEnabled">TRUE</pref>
+                    <pref name="zimbraPrefOutOfOfficeReply">${account1.reply}</pref>
+                    <pref name="zimbraPrefOutOfOfficeStatusAlertOnLogin">TRUE</pref>
+                    <pref name="zimbraPrefOutOfOfficeFromDate">20240827145900Z</pref>
+                    <pref name="zimbraPrefOutOfOfficeUntilDate">20240831145900Z</pref>
+                </ModifyPrefsRequest>
+            </t:request>
+            <t:response>
+                <t:select path="//acct:ModifyPrefsResponse"/>
+            </t:response>
+        </t:test>
+
+        <!-- Test 1: Verify OOO status is inactive after untilDate -->
+        <t:test id="check_ooo_inactive_after_date" required="true" depends="VerifyOOO_UntilDateReached">
```

Comment:
VerifyOOO_UntilDateReached is not defined but used in test- check_ooo_inactive_after_date

Reply:
Renamed